docs/examples-profiling-vtune.md

```diff
@@ -30,13 +30,10 @@ For more information on VTune and the analysis tools it provides see its
 
 ### Turn on VTune support
 
-For JIT profiling with VTune, first build with the `vtune` feature enabled:
-
-```sh
-$ cargo build --features=vtune
-```
-
-Then, enable runtime support based on how you use Wasmtime:
+For JIT profiling with VTune, Wasmtime currently builds with the `vtune` feature
+enabled by default. This ensures the compiled binary understands how to inform
+the `ittapi` library of JIT events. But it must still be enabled at
+runtime--enable runtime support based on how you use Wasmtime:
```

Comment:
hmm, it seems the end of this comment after `--` would need to be removed, as it contains `:` but with no extra indication thereafter?

Reply:
I'll leave it and we can fix it up later if we think of a more clear way to word this.

Reply:
Oh, i misread it as a list entry as well, my bad. Maybe can reformulate slightly then:

```suggestion
the `ittapi` library of JIT events. But it must still be enabled at
runtime with one of the following:
```